about.html

```diff
@@ -0,0 +1,86 @@
+    <title>Lagos Tech Fest.</title>
+    <script src="https://kit.fontawesome.com/97c3dbf8a8.js" crossorigin="anonymous"></script>
+    <script type="module" src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <img id="menu" src="images/menu.png" alt="mobile menu" />
+    </header>
+    <!-- desktop menu -->
+    <header id="desktop-menu">
+      <ul class="navigation">
+        <li><img class="facebook" src="images/facebook2.jpeg" alt="facebook" /></li>
+        <li><img class="twitter" src="images/twitter.png" alt="twitter" /></li>
+        <li><p class="nav-desk">English</p></li>
+        <li><p class="nav-desk">My Page</p></li>
+        <li><p class="nav-desk">Logout</p></li>
+      </ul>
+      <nav class="desk-nav">
+        <div class="div-image-desk">
+          <a href="home.html"><img class="image-desk" src="images/Logo.jpg" alt="image"/></a>
+        </div>
+        <div class="container-desk">
+          <ul class="list-desk">
+            <li><a class="anchor" href="about.html">About</a></li>
+            <li><a class="anchor" href="#">Program</a></li>
+            <li><a class="anchor" href="#">join</a></li>
+            <li><a class="anchor" href="#">News</a></li>
+            <li><a class="point" href="#">Sponsor</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="container">
+      <div class="introduction">
+        <p class="intro-text">"Hello, Tech world!"</p>
+        <h1>Examining the evolution of E-commerce in Africa</h1>
+        <p class="paragraph-title">
+            The event annually provides an unrivaled platform to identify opportunities and plan strategies by having a strong understanding of the investment opportunities in the country’s tech ecosystem. Driving Investment has been at the core of the event strategy, We provide an opportunity to develop strategies based on the latest regulatory developments.
+        </p>
+        <p class="address">For more enquiries about Lagos Tech Fest.</p>
+        <p class="address-2">lagostechfest@techisfuture.org</p>
+      </div>
+    </section>
+    <section class="about">
+      <h4>lagostechfest 2023</h4>
+      <div class="guide"></div>
+      <p class="paragraph-title">Lagos TechFest is a Nigeria-focused tech conference, exhibition, and networking platform that provides an active platform for discussions and deal-making.</p>
+      <!-- <div class="logo-1"> -->
+      <img class="logo" src="images/lagfestlogo.png" alt="lagtechfestlogo" />
+      <!-- </div> -->
+    </section>
+    <section>
+      <h3>Lagos Tech Fest</h3>
+      <div class="guide"></div>
+      <p class="paragraph-title">Checkout out how we held our last two conferences.</p>
+      <div class="image-class">
+        <img src="images/lagtechfesst4.jpeg" alt="2021 outreach" class="outreach img" />
+        <img src="images/lagtechfest5.jpeg" alt="2022 outreach" class="outreach img-1"/>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly add an orange/red overlay over the images of this section and a text centered over them to match the template.
 - Your design:
 - 
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/61919617/225373441-5d937106-f8c0-4a22-8fb2-82409e39a52f.png">

 - Template design:
 - 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/61919617/225373517-383f39ff-3640-4cd0-8547-48a80350b430.png">